src/Docker.DotNet/Credentials.cs

```diff
@@ -1,16 +1,24 @@
 ﻿using System;
 using System.Net.Http;
+using Microsoft.Net.Http.Client;
 
 namespace Docker.DotNet
 {
     public abstract class Credentials : IDisposable
     {
         public abstract bool IsTlsCredentials();
 
+        public abstract bool IsSshCredentials();
+
         public abstract HttpMessageHandler GetHandler(HttpMessageHandler innerHandler);
 
+        public virtual ManagedHandler.StreamOpener GetSshStreamOpener(string username)
+        {
+            return null;
+        }
```

Comment:
> Now that remains in this PR is enough scaffolding for anyone to hook up with their own SSH library.

Did I understand correctly that anyone who would like to connect via SSH must implement this part themselves?